.pre-commit-config.yaml

```diff
@@ -120,11 +118,6 @@ repos:
         entry: "_nxver > |_nxver <= | < _nxver| >= _nxver"
         # Use e.g. `_nxver < ...` or `_nxver >= ...` instead
         exclude: .pre-commit-config.yaml
-  - repo: https://github.com/sphinx-contrib/sphinx-lint
-    rev: v1.0.0
-    hooks:
-      - id: sphinx-lint
-        args: ["--enable=all", "--disable=line-too-long"]
```

Comment:
I'm guessing this just wasn't very helpful.  Is there any other context around removing it?

Reply:
oh great -- that's a nice check